ruff.toml

```diff
@@ -0,0 +1,88 @@
+# automatically generated file, do not modify
+# for ruff version 0.8.1 (or higher)
+# note: 'E241', 'E272', 'E201', 'E221' are ignored on runbot in test files when formating a table like structure (more than two space)
+# some rules present here are not enabled on runbot (yet) but are still adviced to follow when possible : ["B904", "COM812", "EM101", "I001", "TRY002"]
+
+
+target-version = "py310"
+
+[lint]
+preview = true
+select = [
+    "BLE",
+    "C",
+    "COM",
+    "E",
+    "EM",
+    "EXE",
+    "F",
+    "FA",
+    "FLY",
+    "G",
+    "I",
+    "ICN",
+    "INT",
+    "ISC",
+    "LOG",
+    "PGH",
+    "PIE",
+    "PLC",
+    "PLE",
+    "PLW",
+    "PYI",
+    "RUF",
+    "SIM",
+    "SLOT",
+    "T",
+    "TC",
+    "TID",
+    "TRY",
+    "UP",
+    "W",
+    "YTT"
+]
+ignore = [
+    "C408",
+    "C420",
+    "C901",
+    "E266",
+    "E501",
+    "E713",
+    "E741",
+    "EM102",
+    "FA100",
+    "PGH003",
+    "PIE790",
+    "PIE808",
+    "PLC2701",
+    "PLW2901",
+    "RUF001",
+    "RUF005",
+    "RUF012",
+    "RUF021",
```

Comment:
Remove parenthesize-chained-operators?
Just for readability sake.

Reply:
It was asked to be removed by JKE and qsm even if tde and I would  prefer to keep it.
I moved it in the optional list